src/lib/Components/Inbox/Bids/ActiveBid.tsx

```diff
@@ -134,32 +134,13 @@ export default Relay.createContainer(ActiveBid, {
       fragment on LotStanding {
         is_leading_bidder
         active_bid {
-          id
           max_bid {
-            cents
             display
           }
           sale_artwork {
-            lot_label
             lot_number
-            position
             reserve_status
-            counts {
-              bidder_positions
-            }
-            sale {
-              live_start_at
-              end_at
-              is_live_open
-              is_closed
-            }
-            highest_bid {
-              cents
-              display
-            }
             artwork {
-              id
-              title
```

Comment:
Not that I know of, I think it's an isolated case but we can always check other relay containers in the app to make sure we're not over-fetching data.